engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-details.html

```diff
@@ -80,6 +80,13 @@ <h2 translate>{{ contentView.name }}</h2>
         </ul>
       </li>
 
+      <li ng-class="{active: stateIncludes('content-view.repositories.deb')}"
+          ng-hide="contentView.composite">
+        <a ui-sref="content-view.repositories.deb.list" translate>
+          deb Repositories
```

Comment:
I'd prefer this to be consistent with capitalization of the other tabs.  "Deb Repositories"

Reply:
Actually, i am not quite sure where to call them "Apt Repositories".

Reply:
Thats a good point, anytime we refer to the repositories we should likely refer to them as "Apt Repositories", including when creating a new repo.